### PR TITLE
MS-26: Positions - List positions

### PR DESCRIPTION
**Pull request by EugenioPonzone**
Add mock data positions.
Add GET (list)  positionController

### CODE REVIEW

File: controllers/positions.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+
+
+module.exports = {
+  getAll: getAll,
+  getById: getById,
+  getByName: getByName,
+  add: add,
```
Comment (nicolasblasco) [inline controllers/positions.js:79]: Change for: 
create: create,

File: controllers/positions.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+const fs = require('fs');
+const positions = JSON.parse(fs.readFileSync('./data/positions.json'));
+
```
Comment (damianalvarez) [inline controllers/positions.js:3]: @EugenioPonzone can we get rid of the lines 3, 4 and 5?

File: index.js [hunk 3]
```diff
@@ -2,10 +2,35 @@ const express = require('express');
...
+app.get('/positions/byId/:id', positionsController.getById);
+app.get('/positions/byName/:name', positionsController.getByName);
+
 app.listen(port, () => {
-  console.log(`MindSet server listening at http://localhost:${port}`)
-});
\ No newline at end of file
+  console.log(`MindSet server listening at http://localhost:${port}`);
+});
+
```
Comment (damianalvarez) [inline index.js:27]: @EugenioPonzone can we also get rid of the lines 27 to 35?

File: controllers/positions.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+// MS26-POSITIONS - List Positions
+
+const getAll = (req, res) => {
+    res.json(positions);
+};
+
+const getById = (req, res) => {
+    let filteredPosition = positions.find(element => element.id === req.params.id);
```
Comment (damianalvarez) [inline controllers/positions.js:12]: @EugenioPonzone let's use a `const` instead of the `let` here an on the `getByName` method as well

**Review by damianalvarez — CHANGES_REQUESTED** [review]

**Review by damianalvarez — APPROVED** [review]

**Pull request merged**